config/session.js

```diff
@@ -20,6 +20,8 @@ module.exports.session = {
    * of your users, forcing them to log in again.                             *
    *                                                                          *
    ***************************************************************************/
+
+  // TODO this should be hidden from the public repo and moved to an ENV we set at build time
```

Comment:
@itsalaidbacklife this needs your input, we should move it.

Reply:
Chalking this up to a separate issue. Moved to [Issue 70](https://github.com/cuttle-cards/cuttle/issues/70)